src/cuda_xengine.cu

```diff
@@ -130,6 +130,12 @@ __device__ static void operator+=( float4 &a, const float4 b ) {
  a = t;
 }
 
+#ifdef FIXED_POINT
+#define make_float4_rnd(x, y, z, w) make_float4(round(x), round(y), round(z), round(w))
```

Comment:
The programming guide says:
"The recommended way to round a single-precision floating-point operand to an integer, with the result being a single-precision floating-point number is rintf(), not roundf()."
There is discussion here which states that rintf uses HW-supported round to nearest even: https://devtalk.nvidia.com/default/topic/838611/cuda-math-api-difference-between-nearbyintf-roundf-and-rintf-/?offset=1